challenge_9/python/shaowen/unit_test.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```

Comment:
Tks for the suggestion!
I just added, 
I never noticed this before, but in my understanding, this is only good for my own use, right? 
Cus the path is specific for my rig. If others want to run in './blabla.py' way, they have to change to their own path.